### DISCUSSION TITLE
Autogenerate screens option with Details screens

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9436] [Help]
Hi there,

Would it be possible to get the old CRUD screens with /:_id and /new/row back as an option?

Thanks in advance!

**Comment by shogunpurple** [comment]
@ecxs its unlikely we will bring back that option specifically as it creates too may screens and is more complicated for the user. 

What problems were you having with the single screen?

**Reply by ecxs** [reply]
With the introduction of the Details side panel, it's is now easily possible to get a tree of screens that partially disappears below the details panel. See issued https://github.com/Budibase/budibase/discussions/9437

Plus I also still have use cases for details screens.

**Reply by melohagan** [reply]
You can still create those types of CRUD screens as before: https://docs.budibase.com/docs/url-parameters

**Reply by ecxs** [reply]
Thanks @melohagan ... I now, but the autogenerate was so handy, and I was using it all the time ;-)

**Reply by jmlaurenti** [reply]
Hi @melohagan , I agree with @ecxs , side panels are ok for simple objects with limited number of fields, but ability to continue using dedicated Create/Update screens with master details views is definitively a must have... and doing a full copy paste of an existing screen to a new one, and altering all values is more error prone than having the initial canvcas being auto-generated...
I fully understand you have to make clear design choices to keep moving forward, but as long as we cannot expand the side panel to any width, there will be a significant gap between these 2 implementations (CRUD vs Side Panel).
Anyway, thanks for hard work !

JML
